types/indexer.go

```diff
@@ -200,19 +211,33 @@ func (t *TransactionIndexer) heightQuery(condition query.Condition, pagination *
 	return t.getByPrefix(prefixKeyForHeight(height), pagination)
 }
 
-func (t *TransactionIndexer) signerQuery(condition query.Condition, pagination *query.Page) (res []*types.TxResult, total int, err error) {
-	signer, err := hex.DecodeString(condition.Operand.(string))
+func (t *TransactionIndexer) signerQuery(primaryCondition query.Condition, secondaryCondition query.Condition, pagination *query.Page) (res []*types.TxResult, total int, err error) {
```

Comment:
Not actionable: I don't think we'll need any more features here, otherwise I would've suggested variadic params (for v1 stuff).